src/main/java/ca/ryangreen/apigateway/generic/GenericApiGatewayRequest.java

```diff
@@ -35,4 +38,8 @@ public InputStream getBody() {
     public Map<String, String> getHeaders() {
         return headers;
     }
+
+    public Map<String, List<String>> getParameters() {
```

Comment:
Let's call this "queryStringParameters" throughout to keep consistent with APIG terminology